src/cmd/cli/command/commands.go

```diff
@@ -289,6 +289,8 @@ func SetupCommands(ctx context.Context, version string) {
 	mcpCmd.AddCommand(mcpSetupCmd)
 	mcpCmd.AddCommand(mcpServerCmd)
 	mcpSetupCmd.Flags().String("client", "", "MCP setup client (supports: claude, windsurf, cursor, vscode)")
+	mcpServerCmd.Flags().Int("auth-server", 0, "MCP auth server")
```

Comment:
@KevyVo  is this suppose to be a port number?

Reply:
I see it used below. May be update the description: 
```suggestion
	mcpServerCmd.Flags().Int("auth-server", 0, "MCP auth server port")
```

Reply:
Yea good catch, I update this. I must forgot to go back to it.